sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -454,8 +454,8 @@ def to_sql(self) -> str:
     def to_bigquery(
         self,
         job_config: Optional[bigquery.QueryJobConfig] = None,
-        timeout: int = 1800,
-        retry_cadence: int = 10,
+        timeout: Optional[int] = 1800,
+        retry_cadence: Optional[int] = 10,
```

Comment:
I think this will be stomped over by #3505

Reply:
set default to `None`: done